components/MovieList/MovieList.js

```diff
@@ -0,0 +1,7 @@
+import styled from "styled-components";
+
+export default function MovieList() {
+    return(
+        
```

Comment:
Do you already use this component somewhere?

Regardless of that I would always suggest to at least return anything to prevent errors etc., like:
```
<p>no movies available</p>
```